### PR TITLE
Navigation and brand header fixes

### PR DESCRIPTION
**Pull request by kruai**
I fixed some URLs in the sidenav and header, and updated the README. Trying to eliminate references to `insights/platform`.

### CODE REVIEW

File: src/js/App/Sidenav/Navigation.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import { appNavClick, clearActive } from '../../redux/actions';
 import NavigationItem from './NavigationItem';
 
-const basepath = `${document.baseURI}platform/`;
+const basepath = `${document.baseURI}`;
```
Comment (karelhala) [inline src/js/App/Sidenav/Navigation.js]: This is already addressed in https://github.com/RedHatInsights/insights-chrome/pull/185

File: src/js/App/Sidenav/NavigationItem.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { NavItem } from '@patternfly/react-core/dist/esm/components/Nav';
 
-const basepath = `${document.baseURI}platform/`;
+const basepath = `${document.baseURI}`;
```
Comment (karelhala) [inline src/js/App/Sidenav/NavigationItem.js]: This is already been worked on in https://github.com/RedHatInsights/insights-chrome/pull/185

File: src/js/App/Sidenav/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ class SideNav extends Component {
         return (<Fragment>
             <Split className="ins-c-navigation__header">
                 <SplitItem className="ins-c-page__home-icon">
-                    <a href={`${ document.baseURI }platform/landing`}>
+                    <a href={`${ document.baseURI }landing`}>
```
Comment (karelhala) [inline src/js/App/Sidenav/index.js]: This is changed in https://github.com/RedHatInsights/insights-chrome/pull/185

**Comment by kruai** [comment]
I think I'm just going to close this one. I didn't know you were already making these changes and it looks like yours already covers the main issues I was addressing.

**Pull request closed without merging**